package.json

```diff
@@ -18,6 +18,6 @@
   },
   "homepage": "https://github.com/LittleSkinCommspt/commspt-bot-manual#readme",
   "devDependencies": {
-    "vuepress": "^1.5.4"
+    "vuepress": "^2.0.0-beta.45"
```

Comment:
Is this a stable version? I don't think so.

Cubik65536 at #15

> Beta releases should NOT be used in production.

FLYEMOJ1 at #15

> > Beta releases should NOT be used in production.
> Agree with that.

jinzhijie at #15 

> 暂时不考虑升级到非稳定版本

Reply:
So what can we do? Update to VuePress v1.9.x?
Also we need to solve dependencies alert.